UPGRADE-3.0.md

```diff
@@ -1739,6 +1768,28 @@ UPGRADE FROM 2.x to 3.0
    `Process::setInput()` and `Process::getInput()` that works the same way.
  * `Process::setInput()` and `ProcessBuilder::setInput()` do not accept non-scalar types.
 
+### Monolog Bridge
+
+ * `Symfony\Bridge\Monolog\Logger::emerg()` was removed. Use `emergency()` which is PSR-3 compatible.
+ * `Symfony\Bridge\Monolog\Logger::crit()` was removed. Use `critical()` which is PSR-3 compatible.
+ * `Symfony\Bridge\Monolog\Logger::err()` was removed. Use `error()` which is PSR-3 compatible.
+ * `Symfony\Bridge\Monolog\Logger::warn()` was removed. Use `warning()` which is PSR-3 compatible.
+
+ * Removed the feature that allowed finding deep items in `ParameterBag::get()`.
```

Comment:
this has nothing to do with the monolog bridge. you copy/pasted the changes of HttpFoundation below